tests/dummy/config/targets.js

```diff
@@ -6,13 +6,6 @@ const browsers = [
   'last 1 Safari versions'
 ];
 
-const isCI = !!process.env.CI;
-const isProduction = process.env.EMBER_ENV === 'production';
```

Comment:
Why remove this code?

Reply:
There is no need to build the dummy app with IE9 or any other legacy browser support for testing.

Reply:
I do not think it is used anymore? 🤔

Reply:
This just comes from ember-cli blueprints though, right? If so, I would like to keep as much in line with the blueprints as we can.

Reply:
I am a little confused. We do not want to support IE, afaik. If so, that makes sense to remove the code. Am I missing something? Otherwise, both `isCI` and `isProduction` will be unused 🤔

Reply:
https://github.com/html-next/vertical-collection/pull/330

Reply:
Deleting it breaks the tests. Sorry, I do not see a good reason to keep this code around. I am happy to follow up with any change later. I do not think we should block the release on this 😄

Reply:
@twokul can we please just keep it as it was before this PR? It matched the ember-cli blueprints before.

Reply:
I do not understand the reasoning but I have reverted the change as you asked https://github.com/html-next/vertical-collection/pull/321/commits/b0d44dbf9e40d721507321ab124bb5f590947905

Reply:
The tests won't pass since `ember-maybe-import-regenerator` is not a dependency anymore

Reply:
The reasoning is to match the blueprints for ease of future Ember updates and better diffs. I do not disagree with you that the code is not necessary, I just always want to strive to match the current blueprints in all cases, unless we have a reason to diverge.

Reply:
@rwwagner90 we have a reason to diverge, IE11 is not supported. Right?